rali/rali/source/rali_api_augmentation.cpp

```diff
@@ -1736,6 +1775,49 @@ raliRandomCrop(
 }
 
 
+extern "C" RaliImage RALI_API_CALL
+raliSSDRandomCrop(
+        RaliContext p_context,
+        RaliImage p_input,
+        bool is_output,
+        RaliFloatParam p_threshold,
+        RaliFloatParam p_crop_area_factor,
+        RaliFloatParam p_crop_aspect_ratio,
+        RaliFloatParam p_crop_pox_x,
+        RaliFloatParam p_crop_pos_y)
+{
+    Image* output = nullptr;
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Image*>(p_input);
+    auto crop_area_factor  = static_cast<FloatParam*>(p_crop_area_factor);
+    auto crop_aspect_ratio = static_cast<FloatParam*>(p_crop_aspect_ratio);
+    auto x_drift = static_cast<FloatParam*>(p_crop_pox_x);
+    auto y_drift = static_cast<FloatParam*>(p_crop_pos_y);
+
+    try
+    {
+        if(!input || !context)
+            THROW("Null values passed as input")
+        ImageInfo output_info = input->info();
+        output_info.width(input->info().width());
+        output_info.height(input->info().height_single());
+        output = context->master_graph->create_image(output_info, is_output);
+        output->reset_image_roi();
+        std::shared_ptr<RandomCropNode> crop_node =  context->master_graph->add_node<RandomCropNode>({input}, {output});
```

Comment:
Codacy found an issue: [Either the condition '!context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)